CONTRIBUTING.md

```diff
@@ -13,13 +31,26 @@ please head over to our [feedback page](https://feedback.mongodb.com/forums/9308
 Please create a [GitHub issue](https://github.com/mongodb/mongocli/issues/new?assignees=&labels=&template=bug_report.md) describing the kind of problem you're facing
 with as much detail as possible, including things like operating system or anything else may be relevant to the issue.
 
-## Submitting a Patch
+## Submitting Patches
 
-Before submitting a patch to the repo please consider opening an [issue first](#reporting-issues).
+The MongoDB CLI project welcomes all contributors and contributions regardless of skill or experience level. 
+If you are interested in helping with the project, we will help you with your contribution.
 
-### Contributor License Agreement
+### Code Contribution Guidelines
 
-For patches to be accepted, contributors must sign our [CLA](https://www.mongodb.com/legal/contributor-agreement).
+We want to create the best possible product for our users and the best contribution experience for our developers,
+we have a set of guidelines to ensure that all contributions are acceptable.
+
+To make the contribution process as seamless as possible, we ask for the following:
+
+* Go ahead and fork the project and make your changes.  
+  We encourage pull requests to allow for review and discussion of code changes.
+* When you’re ready to create a pull request, be sure to:
+  * Sign the [CLA](https://www.mongodb.com/legal/contributor-agreement).
+  * Have test cases for the new code. If you have questions about how to do this, please ask in your pull request.
+  * Run `make fmt`.
+  * Add documentation if you are adding new features or changing functionality.  
```

Comment:
[nit] Link to the `Maintainer's Guide` section here or explicitly call out that docs needs to be tagged (it's easy to forget and a prompt to the user is very helpful)

Also, do we have a note in the new PR template with that note, so again people see it everytime they make a PR?

Reply:
I'm thinking about this, I don't want to link to maintainer's guide to try and not overwhelm people here we have a field to signal you added docs but may be lost, trying to consider if we can use a github action to add labels or something as a CODEOWNERS may be hard to add for this point

Reply:
No problem - it's a nit, just want to help people be able to figure out what to do as easily as possible.

Reply:
I looked at https://github.com/actions/labeler but I can only filter by files and this may cover flags as we have then centralize but for commands it may be a bit more complex so I'll let this as tbd